sidebarsDocs.js

```diff
@@ -23,528 +23,652 @@ const sidebars = {
+          },
+        ]
+      },
+      {
+        type: 'category',
+        label: 'Working with Plugins',
+        items: [
+          {
+            type: 'doc',
+            id: 'installing-plugins',
+            label: 'Installing Plugins',
+          },
+          {
+            type: 'doc',
+            id: 'plugin-development',
+            label: 'Plugin Development',
+          },
+          {
+            type: 'category',
+            label: 'Management Plugin',
+            link: {type: 'doc', id: 'management'},
+            items: [
+              {
+                type: 'doc',
+                id: 'management-cli',
+                label: 'Management CLI',
+              },
+            ],
+          },
+          {
+            type: 'category',
+            label: 'Federation Plugin',
+            link: {type: 'doc', id: 'federation'},
+            items: [
+              {
+                type: 'doc',
+                id: 'federated-exchanges',
+                label: 'Federated Exchanges',
+              },
+              {
+                type: 'doc',
+                id: 'federated-queues',
+                label: 'Federated Queues',
+              },
+              {
+                type: 'doc',
+                id: 'federation-reference',
+                label: 'Federation Reference',
+              },
+            ],
+          },
+          {
+            type: 'category',
+            label: 'Shovel Plugin',
+            link: {type: 'doc', id: 'shovel'},
+            items: [
+              {
+                type: 'doc',
+                id: 'shovel-static',
+                label: 'Configuring Static Shovels',
+              },
+              {
+                type: 'doc',
+                id: 'shovel-dynamic',
+                label: 'Configuring Dynamic Shovels',
+              },
+            ],
+          }, 
+          {
+            type: 'doc',
+            id: 'stream',
+            label: 'Stream Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'stomp',
+            label: 'STOMP Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'web-stomp',
+            label: 'Web STOMP Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'mqtt',
+            label: 'MQTT Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'web-mqtt',
+            label: 'Web MQTT Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'ldap',
+            label: 'LDAP Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'oauth2',
+            label: 'OAuth 2.0 Authentication/Authorisation Backend Plugin',
+          },    
+        ],  
+      },
+      {
+        type: 'doc',
+        id: 'distributed',
+        label: 'Distributed RabbitMQ Feature',
+      },
+      {
+        type: 'category',
+        label: 'Protocols',
```

Comment:
Hmm, I'm not sure this category should be versioned with the server docs.

Protocols are a general thing that we do not even defined ourselves.

Reply:
ok I will show you this one in a demo of the new TOC on my local environment, I will setup a quick 15 minute session with you tomorrow for that if you are available, thanks.

Reply:
Done, completed updates as a result of our meeting yesterday, thanks